documentation/interactions/uirepresentation.md

```diff
@@ -185,3 +185,29 @@ dependencies {
     add("kspJs", project(":ksp:mutable-ui-processor"))
 }
 ```
+
+## Observing MotionProperty in children UI
+
+Sometimes children UI can depend on the transition animation values. Appyx provides an API to observe `MotionProperty` inside children UI.
+
+In this example, children composable retrieve and display `DpOffset` value from `Position` motion property as well as `Float` value from `RotationY` using this API:  
```

Comment:
```suggestion
## Observing MotionProperty values

Appyx provides an API to observe `MotionProperty` values further down in the composition. This can be useful to build e.g. conditional UI effects based on their values.

This example demonstrates retrieving the animated values of `Position` and `RotationY`: